plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/DebuggerPresenter.java

```diff
@@ -96,7 +96,7 @@ public DebuggerPresenter(
       final @DebuggerToolbar ToolbarPresenter debuggerToolbar,
       final DebuggerManager debuggerManager,
       final WorkspaceAgent workspaceAgent,
-      final DebuggerResourceHandlerFactory resourceHandlerManager,
+      final DebuggerLocationHandlerManager resourceHandlerManager,
```

Comment:
Maybe need to rename variable `resourceHandlerManager` according to class name? WDYT?